cypress/e2e/06_add_building/add_building.cy.js

```diff
@@ -0,0 +1,565 @@
+        cy.wait(2000);
+        cy.get('[aria-label="BU Fusion ID"').click().type('03');
+        cy.wait(2000);
+
+        cy.get('.tabbed-workflow-footer-button-container > .btn-success').contains('Save and Continue').click();
+
+        // Map tab
+        cy.get('.mapboxgl-canvas');
+        cy.get('.workbench-card-sidebar');
+        cy.get('#toggle-basemap-panel-button').contains('Basemap').click();
+        cy.get('[aria-label="Dark"]').click();
+        cy.get('[aria-label="ESRI World"]').click();
+        cy.get('[aria-label="ESRI World Topo"]').click();
+        cy.get('[aria-label="Light"]').click();
+        cy.get('[aria-label="Open Street Map"]').click();
+        cy.get('[aria-label="Satellite Streets"]').click();
+        cy.get('[aria-label="Stamen Terrain"]').click();
+        cy.get('[aria-label="TC-copy"]').click();
+        cy.get('[aria-label="satellite"]').click();
+        cy.get('.active-basemap').click();
+
+        cy.get('#toggle-overlays-panel-button').contains('Overlays').click();
+        cy.get('#toggle-legend-panel-button').contains('Legend').click();
+
+        cy.get('.card_component.feature_shape').contains('Feature Shape').scrollIntoView();
+        cy.get('.select2-selection__rendered').contains('Select an option').click();
+
+        cy.get('.select2-results__option').contains('Approx').click();
+        cy.get('.select2-selection__rendered > .select2-selection__clear').first().click({force: true});
+        cy.get('.select2-results__option').contains('Archaeological Event').click();
+        cy.get('.select2-selection__rendered > .select2-selection__clear').first().click({force: true});
+        cy.get('.select2-results__option').contains('Area').click();
+        cy.get('.select2-selection__rendered > .select2-selection__clear').first().click({force: true});
+        cy.get('.select2-results__option').contains('Dispersed Event').click();
+        cy.get('.select2-selection__rendered > .select2-selection__clear').first().click({force: true});
+        cy.get('.select2-results__option').contains('Linear').click();
+        cy.get('.select2-selection__rendered > .select2-selection__clear').first().click({force: true});
+        cy.get('.select2-results__option').contains('Locality').click();
+        cy.get('.select2-selection__rendered > .select2-selection__clear').first().click({force: true});
+        cy.get('.select2-results__option').contains('Named Loc').click();
+        cy.get('.select2-selection__rendered > .select2-selection__clear').first().click({force: true});
+        cy.get('.select2-results__option').contains('Unknown').click();
+
+        cy.get('.tabbed-workflow-footer-button-container').contains('Save and Continue').click();
+
+        // Digtal Files tab
+        cy.get('.bord-top > .btn').contains('Select Files').click();
+        cy.get('.tabbed-workflow-footer-button-container').contains('Next Step').click()
+
+        // Finish tab
+        cy.wait(2000);
```

Comment:
This tab is just clicking random things. Again names of created persons should be used. The generate HB functionality isnt tested whatsoever